### ISSUE TITLE
 PyTorch 1.3 train accuracy is less than PyTorch 1.0

### ISSUE THREAD

**Issue by ShaneYS** [pytorch/pytorch#29105]
First, I trained pspnet on my own data with pytorch1.0.1.post2 and torchvision0.2.2, and got the F1 score = 0.757.
And then I trained the same case with pytorch1.3.0 and torchvision0.4.1, and got the F1 score = 0.738.

In order to confirm the result， I trained pspnet on another my own dataset:
1. with  pytorch1.0.1.post2 and torchvision0.2.2, F1 = 0.658.
2. with pytorch1.3.0 and torchvision0.4.1, F1 = 0.629.

Where goes wrong? Is this a bug about pytorch version?

**Comment by soumith** [comment]
there's not enough info to tease out.

I'd suggest starting minimizing the variability -- small network, toy dataset.
Also, maybe train a few randomly initialized networks and look at the mean / std of the accuracy. One data point doesn't say enough.

Once you get some more clear data, file a new issue.
